framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -501,9 +502,11 @@ public void cancel() {
           @Override
           public void run() {
             try {
-              Path src = new Path(CWD + "/" + Utils.getDrillTestProperties().get("DRILL_TEST_DATA_DIR"), datasource.src);
+              Path src = new Path(CWD + "/" + Utils.getDrillTestProperties().get("DRILL_TEST_DATA_DIR") + "/" + datasource.src);
               Path dest = new Path(drillTestData, datasource.dest);
-              hdfsCopy(src, dest, false, fsMode);
+
+              LOG.debug("Copy " + src + " to " + dest);
```

Comment:
I think it's better leave the debug log inside the function. If you have multiple calls to the function, you don't need to add multiple LOG lines.